user-guide/Advanced_Modules/DOM/DOM_examples/DOM_BulkProcessing_Example.md

```diff
@@ -6,7 +6,11 @@ uid: DOM_BulkProcessing_Example
 
 From DataMiner 10.4.2/10.5.0 onwards<!-- RN 37891 -->, the `DomInstance` CRUD helper component supports processing multiple `DomInstances` in one call.
 
-This page contains examples on how you can use these calls, provided by the [DomHelper](xref:DomHelper_class#multiple-instances).
+In DataMiner 10.5.0/10.5.2<!-- RN 41546 -->, the error handling has been reviewed. This page contains examples on how you can use these calls, provided by the [DomHelper](xref:DomHelper_class#multiple-instances).
+
+> [!IMPORTANT]
+> In DataMiner versions prior to DataMiner Feature Release 10.5.0/10.5.2<!-- RN 37891 -->, when any validation issue occurs, no exception is thrown when calling the `CreateOrUpdate` or `Delete` methods. Instead, the result of the call should be used to check for which `DomInstances` the call succeeded or failed.
+> The `TryCreateOrUpdate` or `TryDelete` methods are not available in those versions.
```

Comment:
Does "those versions" refer to DataMiner versions prior to 10.5.0/10.5.2 or to something else?

Reply:
Yes. They're getting added in RN41546